samples/next-ab-testing/next.config.js

```diff
@@ -0,0 +1,23 @@
+const ExperimentExtractorPlugin = require("next-experiments/experimentExtractor");
```

Comment:
Can we export this as a named export? `const { ExperimentExtractorPlugin } = require('next-experiments');`?

Reply:
Fixed